processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -106,7 +102,7 @@ public SegmentMetadataQuery(
       @JsonProperty("merge") Boolean merge,
       @JsonProperty("context") Map<String, Object> context,
       @JsonProperty("analysisTypes") EnumSet<AnalysisType> analysisTypes,
-      @JsonProperty("usingDefaultInterval") Boolean useDefaultInterval,
+      @Ignore @JsonProperty("usingDefaultInterval") Boolean useDefaultInterval,
```

Comment:
Please document that this parameter is ignored in a simple comment, also add note that it is going to be removed, and left now only for the sake of compatibility.